README.md

```diff
@@ -1,3 +1,226 @@
-# kessel-sdk-py
+# Kessel SDK for Python
+
+A Python SDK for connecting to Kessel services using gRPC.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Quick Start](#quick-start)
+- [Client Builder](#client-builder)
+- [Configuration](#configuration)
+  - [Target Server](#target-server)
+  - [Credentials](#credentials)
+  - [Keep-Alive Settings](#keep-alive-settings)
+- [Authentication](#authentication)
+- [Examples](#examples)
+- [API Reference](#api-reference)
+- [Error Handling](#error-handling)
+- [Development](#development)
+
+
+## Installation
+
+```bash
+pip install kessel-sdk
+```
+
+## Quick Start
+
+```python
+import grpc
+
+from kessel.inventory.v1beta2 import (
+    ClientBuilder, 
+    rbac, 
+    check_request_pb2,
+    resource_reference_pb2, 
+    reporter_reference_pb2
+)
+
+# Build a stub pointing at a local instance
+stub = (
+    ClientBuilder
+    .with_defaults_localhost(9000)
```

Comment:
I recall the ClientBuilder had multiple `with_defaults_*` calls. Should we just set the defaults with a single call and use other methods to set stuff like insecure, targets, etc.

A pattern I recall from C++/Java is (and unsure if is less idiomatic in python)

```
ClientBuilder
.builder()  # Creates the builder and sets the defaults
.target("localhost:9000") # Or even localhost is set as default and instead we use .port(9000) ?
```

Reply:
Yeah this is something that can be done in python, and makes the python sdk more consistent with the others. I plan to update this in a future release PR 😃